coopr-docs/docs/source/guide/superadmin/shell-automator-plugin.rst

```diff
@@ -74,11 +74,10 @@ How it Works
 ============
 
 A more in-depth look at Shell Automator:
-	1. The source of truth for any custom scripts is the local plugin directory on the provisioner host.  For example using the default packages: ``/opt/coopr/provisioner/daemon/plugins/automators/shell_automator/scripts``.
-	2. During the "bootstrap" action:
+	1. During the "bootstrap" action:
 		a. These scripts are bundled and scp'd to the remote host
 		b. The scripts are extracted on the remote host, by default to ``/var/cache/coopr/scripts``
-	3. When a defined shell service action runs (ie, install, configure, start, etc):
+	2. When a defined shell service action runs (ie, install, configure, start, etc):
```

Comment:
Do you want e.g. here (for instance, one of those 3) instead of ie (in other words ...)?